components/AllocatedCases/AllocatedCases.jsx

```diff
@@ -0,0 +1,69 @@
+import { useState, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import AllocatedCasesTable from 'components/AllocatedCases/AllocatedCasesTable';
+import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import { getWorkers, allocatedCases } from 'utils/api/allocatedWorkers';
+import Spinner from 'components/Spinner/Spinner';
+
+const AllocatedCases = ({ id }) => {
+  const [error, setError] = useState();
+  const [loading, setLoading] = useState();
+  const [allocWorkers, setAllocWorkers] = useState();
+  const [allocCases, setAllocCases] = useState();
+  const getData = useCallback(async () => {
+    try {
+      const workData = await getWorkers();
+      const casesData = await allocatedCases();
```

Comment:
`allocatedCases` is blocked by `getWorkers`, I would move this BE side and merge it in one call (i.e. `/api/workers/:id`)